web-app/src/components/tournament/creator/TournamentTypeSelectorItem.vue

```diff
@@ -1,14 +1,14 @@
 <template>
   <div class="row col-8 col-sm-4 justify-center q-ma-md">
-    <q-btn class="row col-12" no-caps padding="none">
+    <q-btn class="row col-12 selectorButton" no-caps padding="none">
       <q-card
         flat
-        class="col-12 tournamentTypeCard"
+        class="row col-12 tournamentTypeCard items-center"
         :class="tournamentType === activeType ? 'activeTypeCard' : ''"
         @click="updateTournamentType"
       >
```

Comment:
When did you set height/width through `selectorButton` to `q-btn` is required to set this same to child component `q-card`?

Reply:
When not set, q-card doesn't take full btn space, pic below 
![obraz](https://user-images.githubusercontent.com/9595999/120507313-fc5fa680-c3c6-11eb-8b38-7e41d8c2798d.png)